package.json

```diff
@@ -71,9 +71,9 @@
   "dependencies": {
     "@babel/core": "^7.18.10",
     "@babel/preset-env": "^7.18.10",
```

Comment:
I added this package using the command:

`npm install parse-community/node-apn`

I originally tried this from node version 18* but it added a bunch of changes to the file unrelated to my update (adding the prefix "node_modules" to each depedency). So I downgraded my node to an earlier version: v14.21.3 per some advice from [SO](https://stackoverflow.com/questions/65783740/why-does-npm-install-prefix-my-packages-with-node-modules-in-my-package-loc)